widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1537,6 +1537,19 @@ button#schedule-dialog-select-button {
 }
 /* endregion GWT DialogBox, PromptDialogBox RESPONSIVE */
 
+/* region Responsive Properties dialog - Start */
+.pentaho-dialog.responsive.properties-dialog .pentaho-tab-deck-panel > div > * {
```

Comment:
From this point on, the File Properties dialog will always be responsive.
Do you need the `.pentaho-dialog.responsive` to increase the CSS specificity of the rule? Otherwise, I think it's best to just use the `.properties-dialog` class.

Reply:
Oh no, you are correct, globalRuby would always have responsive elements only. Got it, would change it

Reply:
addressed